### ISSUE TITLE
scylla_setup: virtio-blk device is not listed on interactive RAID setup

### ISSUE THREAD

**Issue by syuu1228** [scylladb/scylla#4066]
Since we only listing /dev/sd*, /dev/hd*, /dev/xvd*, /dev/nvme*, /dev/mapper/*, virtio-blk device does not appeared (it's /dev/vd*).
We probably won't see it on IaaS instances but user may use it on local VM.

**Comment by avikivity** [comment]
Try: `lsblk  -lpndo name`

Another alternative is to walk through /sys/block and filter out what we don't need.

**Comment by syuu1228** [comment]
@avikivity I found that my previous comment was wrong, we actually uses lsblk when lsblk supported "-p" option, otherwise it will just use the glob pattern that I previously commented.
(lsblk -p is not supported on Ubuntu 14, we needed to verify it: #1030)

The script tries "if lsblk --help | grep -q -e '^\s*-p'; then" in Python, but I just realized the code has a bug, always fails to use lsblk and just using the glob pattern.
https://github.com/scylladb/scylla/blob/master/dist/common/scripts/scylla_setup#L79

But since -p option is just for getting full path of block devices, we probably still able to run "lsblk -nr" and append "/dev/" on the result, should be work on Ubuntu 14.
